float32 gaze_x
float32 gaze_y
float32 gaze_z
---
bool success
